target/linux/ramips/base-files/lib/preinit/04_set_netdev_label

```diff
@@ -0,0 +1,15 @@
+set_netdev_labels() {
+	local dir
+	local label
+	local netdev
+
+	for dir in /sys/class/net/*; do
+		[ -r "$dir/of_node/label" ] || continue
+		IFS= read -rd '' label < "$dir/of_node/label"
+		netdev="${dir##*/}"
+		[ "$netdev" = "$label" ] && continue
+		ip link set "$netdev" name "$label"
+	done
+}
+
+boot_hook_add preinit_main set_netdev_labels
```

Comment:
I guess this file can be moved to subtarget directory `target/linux/ramips/mt7621/xxx`.

Reply:
Right, all the other subtargets are using the old Ralink driver anyway.